src/browser/declarativewebutils.cpp

```diff
@@ -152,13 +149,6 @@ void DeclarativeWebUtils::updateWebEngineSettings()
     webEngineSettings->setPreference(QString("keyword.enabled"), QVariant(true));
 
     setRenderingPreferences();
-
-    // Disable SSLv3
-    webEngineSettings->setPreference(QString("security.tls.version.min"), QVariant(1));
```

Comment:
It doesn't affect this PR, but I notice FF 78 was due to disable support for TLS 1.0 and 1.1. So this may be something to revisit in the not-too-distant future: https://hacks.mozilla.org/2020/02/its-the-boot-for-tls-1-0-and-tls-1-1/

Reply:
Thanks. We should adjust embedding.js accordingly.

Reply:
Created JB#50522